clients/feeder/feeder.go

```diff
@@ -163,7 +163,7 @@ func handleNotFound(dir, queryArg string, w http.ResponseWriter) {
 	if dir == "transaction" && queryArg == "transactionHash" {
 		w.Write([]byte("{\"finality_status\": \"NOT_RECEIVED\", \"status\": \"NOT_RECEIVED\"}")) //nolint:errcheck
 	} else {
-		w.WriteHeader(http.StatusNotFound)
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
I don't think this change is relevant to this PR. Please correct me if I'm wrong.

Reply:
I found it relevant for test cases (as the feeder endpoint didn't exist on mainnet) to simulate real scenarios, I changed it to `StatusBadRequest` as the real feeder endpoint returns that status in case of an unexisting endpoint